lib/pivotaly/createPTStatusBarItem.spec.js

```diff
@@ -0,0 +1,12 @@
+const {createPTStatusBarItem} = require('./createPTStatusBarItem')
+
+describe('#createPTStatusBarItem', () => {
+  test('test create status bar object', () => {
+    expect(typeof createPTStatusBarItem()).toBe('object')
```

Comment:
I think this has undesired side effects. Can we mock vscode first?

Reply:
@brayovsky all tests are using a mocked `vscode`. I don't need to explicitly call the mock at the top of the file

Reply:
@e-liyai 👍